google/ggrc-core
# GGRC-8194:Hint of the "Default Assessment Type" column contains object names in plural what makes import failed

# Issue description

*Hint of the "Default Assessment Type" column contains object names in plural what makes import failed*

# Steps to test the changes

1.Export "Assessment Template" template
2.Fill in the required columns to create new Assessment Template object
3.Check the hint of the "Default Assessment Type" column
4.Copy-paste any object name into "Default Assessment Type" column
5.Import a file
Actual result: Import successful. 

# Solution description

*Change default values dict.*

# Sanity checklist




# PR Review checklist
